client/src/utils/format.js

```diff
@@ -10,6 +10,10 @@ export function format(x) {
       x.entries(),
       ([k, v]) => `${k} => ${v}`
     ).join(', ')}})`;
+  } else if (typeof x === 'bigint') {
+    return x.toString() + 'n';
+  } else if (typeof x === 'symbol') {
+    return Symbol('Sym').toString();
```

Comment:
```suggestion
    return x.toString();
```
Otherwise things like `Symbol('foo')` would also be formatted to `Symbol('Sym')`

Reply:
Forgot it, sorry for that